src/coap_tinydtls.c

```diff
@@ -23,6 +23,22 @@
 #include <dtls.h>
 #include <dtls_debug.h>
 
+typedef struct coap_tiny_context_t {
+  struct dtls_context_t *d_context;
```

Comment:
I would like to make the field names a bit more descriptive, i.e. `dtls_context` and `coap_context` instead of `d_context` and `c_context`, respectively.

Reply:
Updated